projects/storefrontapp/src/app/spartacus/features/organization/organization-administration-feature.module.ts

```diff
@@ -14,10 +15,16 @@ import {
   AdministrationRootModule,
   ORGANIZATION_ADMINISTRATION_FEATURE,
 } from '@spartacus/organization/administration/root';
+import { environment } from '../../../../environments/environment';
 
+const extensions: Type<any>[] = [];
+
+if (environment.cdc) {
+  extensions.push(CdcAdministrationModule);
```

Comment:
CdcAdministrationModule should be lazy loaded, so it should be imported inside `AdministrationWrapperModule`. Not in the root injector